eclipse-scout-core/src/form/fields/ValueField.js

```diff
@@ -389,6 +395,17 @@ export default class ValueField extends FormField {
     // NOP
   }
 
+  _collectDefaultMenuTypes(menuTypes) {
```

Comment:
Why is this collector necessary? Can't you just initialize it in the constructor?